Tests/Functional/MissingAllTestsOverride/main.swift

```diff
@@ -0,0 +1,22 @@
+// RUN: %{swiftc} %s -o %{built_tests_dir}/MissingAllTestsOverride
+// RUN: %{built_tests_dir}/MissingAllTestsOverride > %t || true
+// RUN: %{xctest_checker} %t %s
+// CHECK: Test Case 'MissingAllTestsOverride.missingAllTestsOverride' started.
+// CHECK: <EXPR>:0: unexpected error: MissingAllTestsOverride.missingAllTestsOverride : threw error "This test case class must override the allTests property" - 
```

Comment:
Aha, neat. Perhaps it would be cool to explore populating those with file and line numbers somehow. A discussion for another time, to be sure!